src/dto.js

```diff
@@ -21,52 +20,52 @@ class BaseDTO {
    * @throws MappingError if mapping is missing
    * @throws MissingPropertyError if a property is missing
    */
-  constructor(data, mapping) {
-    const mappingKeys = Object.keys(mapping || {});
+  constructor (data, mapping) {
+    const mappingKeys = Object.keys(mapping || {})
 
-    if(mappingKeys.length === 0) {
-      throw new MappingError('Mapping required');
+    if (mappingKeys.length === 0) {
+      throw new MappingError('Mapping required')
     }
 
-    mappingKeys.forEach(function(k) {
-      const options = mapping[k].options || {};
-      const key = options.key || k;
-      let value;
+    mappingKeys.forEach(function (k) {
+      const options = mapping[k].options || {}
+      const key = options.key || k
+      let value
 
-      if(options.callback) {
-        value = options.callback(data);
+      if (options.callback) {
+        value = options.callback(data)
       }
-      if(value === undefined) {
-        value = data[key] !== undefined ? data[key] : options.default;
+      if (value === undefined) {
+        value = data[key] !== undefined ? data[key] : options.default
       }
 
-      if(value === undefined) {
-        throw new MissingPropertyError('Required property "' + key + '" is missing');
+      if (value === undefined) {
+        throw new MissingPropertyError('Required property "' + key + '" is missing')
       }
 
-      this[k] = mapping[k].fn(value);
-    }, this);
+      this[k] = mapping[k].fn(value)
+    }, this)
 
-    this.__RAW__ = data;
+    this.__RAW__ = data
   }
 
   /**
    * Returns mapped values as an own object.
    *
    * @returns {Object}
    */
-  toJSON() {
-    const values = {};
+  toJSON () {
+    const values = {}
 
     Object.keys(this)
-      .filter(function(key) {
-        return key !== '__RAW__';
+      .filter(function (key) {
```

Comment:
maybe use arrow functions across the board?

Reply:
we can, but I'd prefer a separate PR for that